### DISCUSSION TITLE
Suddenly 504 Error of MongoDB even on older deployments

### DISCUSSION THREAD

**Discussion by edanum** [neuefische/questions#325] [Web] [answered]
Hey there, 
I am facing some serious problems with my capstone project at the moment. I worked on following PR: https://github.com/edanum/capstone-communifi/pull/56 .  After pushing the PR, I noticed that the Deployment (https://capstone-communifi-df3eajcfw-edanum.vercel.app/login) ended in a login-loop (even if it was working fine at localhost. Maybe I got some settings wrong with my next-auth-git-provider). But now comes the tough part. From there on, every deployment isnt working anymore. The server alsways times out:

![image](https://user-images.githubusercontent.com/79711003/203112092-b68c6efe-f8a5-444e-b83a-271e962ee8bc.png)

This happes on the branch, which is linked above, on my main and even on older branches. It seams like something is wrong with the database, since it always times out. From time to time, I get some data:

https://user-images.githubusercontent.com/79711003/203113597-b9e10d9a-881b-46ac-ba21-3c9a47137c23.mov

I would love to get some quick help, since the end of the capstpone-phase is on wednesday :/

**Comment by Kheirah** [comment, marked as answer]
We have resolved this issue in person.

Due to the infinite loop behind next auth (which is a serverless function on vercel) the total execution time hit a limit on your account and the account was temporarily suspended. Solution: Don't write infinite loops.
